.eslintrc.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
       },
     }],
     "@typescript-eslint/no-use-before-define": ["off"],
-    "@typescript-eslint/semi": ["error", "never"]
+    "@typescript-eslint/semi": ["error", "never"],
+    "@typescript-eslint/ban-ts-comment": 1
```

Comment:
What does the value `1` mean here? Online I found that you can set this to `"off"` and that turns it off?

Reply:
`0` means off, `1` means warn, and `2` means error. We might be able to `@typescript-eslint/eslint-plugin` to a newer version to get something a bit more explicit. The newer interface is `bool` or `"allow-with-description"` that makes you write down why.